runtime/metricsview/executor_rewrite_time.go

```diff
@@ -41,6 +73,47 @@ func (e *Executor) resolveTimeRange(ctx context.Context, tr *TimeRange, tz *time
 		return nil
 	}
 
+	if tr.RillTime == "" {
+		return e.resolveTimeRangeLegacy(ctx, tr, tz, executionTime)
+	}
```

Comment:
This gets called if `tr.Start` or `tr.End` are set – I wouldn't really consider that legacy behavior (which entails we will be deprecating it), it's something we will probably always support

Reply:
Actually it also doesn't handle the error case where e.g. both `Start` and `RillTime` is specified

Reply:
Update